worker/src/main/java/gov/cms/ab2d/worker/processor/PatientClaimsProcessorImpl.java

```diff
@@ -92,6 +92,7 @@
             } else {
                 sinceTime = request.getSinceTime();
             }
+            BFDClient.BFD_BULK_JOB_ID.set(request.getJob());
```

Comment:
Why not add as parameter to method instead?

Reply:
I felt that adding parameters everywhere was too disruptive to the code base.  This is a cross-cutting concern, not actually needed by us for the call, more as an aid for BFD to do analysis of their traffic.